containers/Containerfile

```diff
@@ -44,10 +37,8 @@ RUN yum install --disablerepo=updates -y \
     libdnf-devel \
     libnotify-devel \
     make \
-    npm \
```

Comment:
This breaks the tito tests: 
.tito/cockpit-build.sh: line 12: npm: command not found

Reply:
Wow, you are right.